src/FlameScans/FlameScans.ts

```diff
@@ -163,6 +195,23 @@ export class FlameScans extends Source {
         return time
     }
 
+    override getCloudflareBypassRequest(): Request {
+        return createRequestObject({
+            url: FS_DOMAIN,
+            method: 'GET',
+            headers: {
+                'user-agent': userAgent,
+                'referer': `${FS_DOMAIN}.`
```

Comment:
Should be `${FS_DOMAIN}/` not `${FS_DOMAIN}.`

Reply:
Thanks for the catch. No idea how that happened, though, because this was 95% copy-paste.